pkgs/development/ocaml-modules/decompress/default.nix

```diff
@@ -1,23 +1,23 @@
 { lib, fetchurl, buildDunePackage, ocaml
-, checkseum, bigarray-compat, optint, cmdliner
+, checkseum, optint, cmdliner
 , bigstringaf, alcotest, camlzip, base64, ctypes, fmt
 }:
 
 buildDunePackage rec {
-  version = "1.4.2";
+  version = "1.4.3";
```

Comment:
Version `1.5.1` (28 days ago) is out. Any reason to not use it?

Reply:
At `decompress`,  [opam file](https://github.com/ocaml/opam-repository/blob/master/packages/decompress/decompress.1.5.1/opam#L19) has:
* "crowbar"  {with-test & >= "0.2"}
* "rresult"     {with-test}

And it is not listed in `checkInputs`.